C/plugins/north/OMF/include/omflinkeddata.h

```diff
@@ -0,0 +1,104 @@
+#ifndef OMFLINKEDDATA_H
+#define OMFLINKEDDATA_H
+/*
+ * Fledge OSIsoft OMF interface to PI Server.
+ *
+ * Copyright (c) 2022 Dianomic Systems
+ *
+ * Released under the Apache 2.0 Licence
+ *
+ * Author: Mark Riddoch
+ */
+
+#include <map>
+#include <reading.h>
+#include <OMFHint.h>
+
+/**
+ * The OMFLinkedData class.
+ * A reading is formatted with OMF specifications using the linked
+ * type creation scheme supported for OMF Version 1.2 onwards.
+ *
+ * This is based on the new mechanism discussed at AvevaWorld 2022 and
+ * the mechanism is detail in the Google Doc,
```

Comment:
"the mechanism is detailed in the Google Doc." Is the document visible to the public?